lib/ui/screens/create_post/create_post_controller_impl.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:medi_support/ui/screens/create_post/create_post_controller.dart';
+import 'package:medi_support/ui/screens/create_post/create_post_model.dart';
+import 'package:medi_support/ui/screens/create_post/services/create_post_navigation_service.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'create_post_controller_impl.g.dart';
+
+@riverpod
+class CreatePostControllerImpl extends _$CreatePostControllerImpl
+    implements CreatePostController {
+  @override
+  CreatePostModel build(
+      {required CreatePostNavigationService navigationService}) {
+    CreatePostModel model = const CreatePostModel(
+      title: '',
+      body: '',
```

Comment:
**issue:** These fields need to either be optinal or nullable and initialiszed with one of the values.

Reply:
check if better now